content/wiki/autotest-integration.md

```diff
@@ -55,7 +55,7 @@ autotest-all: --color
 
 Please see [Running Features](/cucumber/running-features/) for more information about profiles, and the [Ruby on Rails](/implementations/ruby/ruby-on-rails/) page for extra help when running in that context.
 
-If you get an error like `When using several—format options only one of them can be without a file (RuntimeError)` when running Features with autospec, you are probably accidentally requiring `spec`. Read about the solution on [[Troubleshooting]].
+If you get an error like `When using several—format options only one of them can be without a file (RuntimeError)` when running Features with autospec, you are probably accidentally requiring `spec`. Read about the solution on Troubleshooting.
```

Comment:
Can we link to specific anchors in the HTML page? Troubleshooting is an h3 heading.

Reply:
Thanks @olleolleolle! Yes, I've added the link; there's no information there though, so it might actually be better to remove both?